contexts/governanceVotes.tsx

```diff
@@ -38,8 +38,9 @@ export const GovernanceVotesProvider: React.FC<ProviderProps> = ({
 
   const [delegatedBefore, setDelegatedBefore] = useState('');
   const { data: voteAmount, isFetching: isLoadingVotes } = useContractRead({
-    address: daoInfo.config.DAO_DELEGATE_CONTRACT,
-    abi: daoInfo.ABI,
+    address:
+      daoInfo.config.DAO_TOKEN_CONTRACT || daoInfo.config.DAO_DELEGATE_CONTRACT,
```

Comment:
Token and delegation are two different functionality even if they address is the same. This is going to cause bugs in the future. Why are you doing || ?

Reply:
This won't bug the application because for votes amount it will be looking DAO_TOKEN_CONTRACT first, if doesn't exist, it will see DAO_DELEGATE_CONTRACT.

|| works like a or (if first is falsy, it will get the second). So if a DAO doesn't have a DAO_TOKEN_CONTRACT, it will get DAO_DELEGATE_CONTRACT.

Would be the same thing like 
```ts
daoInfo.config.DAO_DELEGATE_CONTRACT ? daoInfo.config.DAO_DELEGATE_CONTRACT : daoInfo.config.DAO_TOKEN_CONTRACT
```
or
```ts
DAO_DELEGATE_MODE === 'snapshot' ? daoInfo.config.DAO_TOKEN_CONTRACT : daoInfo.config.DAO_DELEGATE_CONTRACT
```